test/unit/vl-link.test.html

```diff
@@ -39,12 +39,12 @@
 
       test('krijgt de juiste class wanneer de link een attribuut bevat', () => {
         const link = fixture('vl-link-fixture');
-        ['block'].forEach((attribuut) => {
-          assert.isFalse(link.classList.contains('vl-link--' + attribuut));
-          link.setAttribute(`data-vl-${attribuut}`, '');
-          assert.isTrue(link.classList.contains('vl-link--' + attribuut));
-          link.removeAttribute(`data-vl-${attribuut}`);
-          assert.isFalse(link.classList.contains('vl-link--' + attribuut));
+        ['block'].forEach((attribute) => {
+          assert.isFalse(link.classList.contains('vl-link--' + attribute));
+          link.setAttribute(`data-vl-${attribute}`, '');
+          assert.isTrue(link.classList.contains('vl-link--' + attribute));
+          link.removeAttribute(`data-vl-${attribute}`);
+          assert.isFalse(link.classList.contains('vl-link--' + attribute));
```

Comment:
ne forEach met ene in, dan zou k precies gene loop maken ...

Reply:
De for loops stonden toen vermoedelijk in promotie, maar ik zal het aanpassen.